shoop/core/models/_service_behavior.py

```diff
@@ -164,3 +167,42 @@ def get_unavailability_reasons(self, service, source):
         groups_to_match = set(self.groups.all().values_list("pk", flat=True))
         if not bool(customer_groups & groups_to_match):
             yield ValidationError(_("Service is not available for any of the customers groups."))
+
+
+class RoundingBehavior(Enum):
+    ROUND_UP = 1
+    ROUND_DOWN = 2
+
+    class Labels:
+        ROUND_UP = _("round up")
+        ROUND_DOWN = _("round down")
+
+
+class RoundingBehaviorComponent(ServiceBehaviorComponent):
+    name = _("Rounding")
+    help_text = _("Round total order price to the nearest five cents.")
+
+    rounding_behavior = EnumIntegerField(
+        RoundingBehavior,
+        default=RoundingBehavior.ROUND_DOWN,
+        verbose_name=_("rounding behavior"))
+
```

Comment:
This behavior component should probably not go in core, but should be provided by the AppConfig.